### ISSUE TITLE
Task3 - Documentation for re-run models previously created 

### ISSUE THREAD

**Issue by Goretx** [mar-muel/artificial-self-AMLD-2020#3]
Hello, it would be very great for beginners like me to have in the documentation some instruction to re-run a model that has been saved on Drive, either on colab or locally.

Thanks a lot in advance!

**Comment by Goretx** [comment]
I've worked (slowly) on this from Colab and this is what I did:

Clone repo from Github
`!git clone https://github.com/mar-muel/artificial-self-AMLD-2020.git`

Go to the folder of the 3rd project
`%cd /content/artificial-self-AMLD-2020/3`

Install requirements
`!pip install -r requirements.txt`

Mount Google Drive
`from google.colab import drive
drive.mount('/content/drive')`

Create a directory where the models need to be stored
`!mkdir runs`

copy the model from Drive to /runs
`!cp -avr /content/drive/My\ Drive/YOUR-FOLDER/run4 /content/artificial-self-AMLD-2020/3/runs`

interact with the model
`!python interact.py --run_name run4`

Problem is that when I run the interact.py the cell never asks me input. Or, to be more precise: it waits for the input but there's no way for me to do it. Same problem running everything locally over Jupyer Notebook. Is there a problem with the input()?

In other words, it waits for the input without displaying the input box, so there's no wait to write:
![Schermata 2020-07-16 alle 13 44 46](https://user-images.githubusercontent.com/49199091/87667411-921c2400-c76a-11ea-9f2b-6ecb8761fd65.png)

how it should be:
![Schermata 2020-07-16 alle 13 41 39](https://user-images.githubusercontent.com/49199091/87667602-f212ca80-c76a-11ea-83a1-d402b78c6679.png)

Is there a problem with the script or the problem is colab/jupyter? Or am I doing something terribly wrong?

**Comment by mar-muel** [comment]
Hey - Not sure, but just to make sure: can you deactivate any kind of anti-virus/ad block of your browser? 

Cheers

**Comment by Goretx** [comment]
Thanks for the quick reply. I deactivated my ad-block but it gives me the same result. Note that if I run a new cell with a input(), the input box appears normally (like in the second image). It doesn't do that only with the input inside the script.
I'm very confused. I thought that the problem was colab but the same thing happens even when I work locally on jupyter lab.
